frontend/lib/utils/route_util.dart

```diff
@@ -32,6 +34,8 @@ class RouteGenerator {
         return PageTransition(child: SearchPage(), type: PageTransitionType.fade, alignment: Alignment.center);
       case "settingsPrivacy":
```

Comment:
Don't use CamelCase (only fpr Classes and Methods), use lowercase_with_underscore for files, keys and case variables